python/TestHarness/TestHarness.py

```diff
@@ -789,6 +804,18 @@ def parseCLArgs(self, argv):
         tmp_str = getattr(self.options, key)
         setattr(self.options, key, value[0])
 
+    # If attempting to test only failed_tests, open the .failed_tests file and create a list object
+    self.test_list = []
+    if self.options.failed_tests:
+      with open(os.path.join(os.getcwd(), '.failed_tests'), 'r') as tmp_failFile:
+        self.test_list = tmp_failFile.read().split('\n')
+    else:
+      try:
+        self.writeFailedTest = open(os.path.join(os.getcwd(), '.failed_tests'), 'w')
+      except IOError:
+        print 'ERROR: Unable to open', os.path.join(os.getcwd(), '.failed_tests'), 'for writing'
```

Comment:
Hmm, this is the right place for try/catch but it's debatable that this should be a fatal error. I think it's fine for now because if you can't write to this directory, it's highly unlikely that the tests are going to pass when they start writing output files to each of their directories.